### PR TITLE
chore: analytics seeder testing

### PR DESCRIPTION
**Pull request by andreacappadona17**
[comment]: <> (This file has been added on OGCIO fork)
### Ticket:

- None

### Description

Please include a summary of the changes and the related issue.

## Type

- [ ] **Dependency upgrade**
- [ ] **Bug fix**
- [ ] **New feature**
- [ ] **Dev change**

### Checklist:

- [ ] I have added tests that prove my fix is effective or that my feature works

### Screenshots:

N/A

### CODE REVIEW

File: packages/cli/src/commands/database/ogcio/ogcio-seeder-testing.json [hunk 1]
```diff
@@ -283,6 +333,16 @@
         "is_third_party": false,
         "apply_management_api_role": true
       },
+      {
+        "name": "M2M Analytics API",
+        "description": "Machine 2 Machine application used to communicate with the Analytics APIs",
+        "type": "MachineToMachine",
+        "redirect_uri": "",
+        "logout_redirect_uri": "",
```
Comment (mcastellin) [inline packages/cli/src/commands/database/ogcio/ogcio-seeder-testing.json:341]: I think these two can be omitted for M2M apps

**Review by mcastellin — APPROVED** [review]
One minor comment

File: packages/cli/src/commands/database/ogcio/ogcio-seeder-testing.json (hunk 1 above)
Comment (andreacappadona17) [reply packages/cli/src/commands/database/ogcio/ogcio-seeder-testing.json:341]: I think we have them for all of the other as well, so we can keep them for now for consistency and remove all of them in a different PR

**Review by nnorbert — APPROVED** [review]
LGTM!

**Pull request merged**